app/src/main/java/org/oppia/app/fragment/FragmentComponent.kt

```diff
@@ -38,4 +39,5 @@ interface FragmentComponent {
   fun inject(topicPlayFragment: TopicPlayFragment)
   fun inject(topicReviewFragment: TopicReviewFragment)
   fun inject(topicTrainFragment: TopicTrainFragment)
+  fun inject(contentListFragment: ContentListFragment)
```

Comment:
Nit: Try to keep the alphabetical order intact.

Reply:
corrected.